stories/topics/proc-gcp-additional-configs-replace-tls-certificate.adoc

```diff
@@ -0,0 +1,23 @@
+[id="proc-gcp-additional-configs-replace-tls-certificate"]
+
+= Replacing {ControllerName} and {HubName} VM instances SSL/TLS certificate and key
+
+By default, VM instances are secured with a self-signed SSL/TLS certificate with a validity period of ten years.
+Once the certificate expires or you want VM instances to use your own certificate, you are required to replace the SSL/TLS certificate and key.
+
+.Procedure
+. Locate the correct secret for the {PlatformNameShort} deployment by opening the GCP Secrets Manager console and selecting the secret with the name _<deployment_name>-pulp_cert_.
+. Select *NEW VERSION* to add a new version.
+. Enter new SSL/TLS certificate value.
+. Check *Disable all past versions*.
+. Click btn:[ADD NEW VERSION].
+. Select the secret with the name _<deployment_name>-pulp_key_.
+. Select *NEW VERSION* to add a new version.
+. Enter new SSL/TLS key value.
+. Check *Disable all past versions*.
```

Comment:
As before, do you need to do this twice? Are these checkboxes on the same page?

Reply:
Two different secrets.  

<img width="1063" alt="image" src="https://github.com/ansible/aap-docs/assets/60555139/285d4b68-942d-4965-bfc0-e4c1c3dd8474">